### PR TITLE
Updated Archive Concordance script

### PR DESCRIPTION
**Pull request by BenjaminsM**
* Removed arguments from script
* Check on which cluster the script is running and use/source  that config file from the automated folder
* Archive VCF's older than 31 days
*Removed script, from script folder and added to the Automated folder

Testing:

If you want to test this do the following:
* Create test enviroment:
    -Create test folder(s)
    - Copy some old VCF files to test location (rsync/copy and keep timestamp),

* Change paths in the script to your test location 
*Start a cronjob or run this script manually

### CODE REVIEW

File: automated/bin/Archive_Concordance.sh [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+# source config files
+
+source ${EBROOTGAP}/automated/etc/${HOST}.cfg
+
+#Checking if VCF File array is older then ArchiveTime and archive file if this is the case
+
+for i in $(find "/groups/umcg-gd/${TMP_LFS}/Concordance/array/" -type f -maxdepth 1 -mtime +31 -print)
+do
+echo "moving ${i} to the neverUsed folder since 31 days has passed and the Concordance has not been executed yet" >> /groups/umcg-gd/${TMP_LFS}/logs/Archive_Concordance_log.txt
```
Comment (pneerincx) [inline automated/bin/Archive_Concordance.sh]: Small issue, but indentation appears different in the first for loop as compare to the second one.

File: automated/bin/Archive_Concordance.sh [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+# identify on which cluster this script will be executed
+HOST=$(hostname -s)
+
+# source config files
+
+source ${EBROOTGAP}/automated/etc/${HOST}.cfg
+
+#Checking if VCF File array is older then ArchiveTime and archive file if this is the case
+
+for i in $(find "/groups/umcg-gd/${TMP_LFS}/Concordance/array/" -type f -maxdepth 1 -mtime +31 -print)
```
Comment (pneerincx) [inline automated/bin/Archive_Concordance.sh]: Is het necessary to use the hard-coded group name or can this also be fetched from a config file?

Comment (RoanKanninga) [inline automated/bin/Archive_Concordance.sh]: what is this -print doing?

File: automated/bin/Archive_Concordance.sh [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+
+set -e
+set -u
+
+# identify on which cluster this script will be executed
+HOST=$(hostname -s)
+
+# source config files
+
+source ${EBROOTGAP}/automated/etc/${HOST}.cfg
```
Comment (RoanKanninga) [inline automated/bin/Archive_Concordance.sh]: missing quotes

File: automated/bin/Archive_Concordance.sh (hunk 2 above)
Comment (RoanKanninga) [reply automated/bin/Archive_Concordance.sh]: I think the issue is that this is executed by the gap-ateambot user, you can of course put in a configfile ;)

Comment (pneerincx) [reply automated/bin/Archive_Concordance.sh]: What it should do and what you can lookup in the man pages for the ```find``` command ;)
This is the better and safer option compared to an ```ls```.

**Pull request merged**